app/(auth)/index.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  View,
+  Text,
+  Pressable,
+  ImageBackground,
+  StatusBar,
+} from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { Carousel } from "@/components/Carousel";
+import { PressableLink } from "@/components/PressableLink";
+import { CarouselData } from "@/types/carouselData";
+
+const carouselData: CarouselData[] = [
+  {
+    title: "Welcome to Rookies 👋",
+    image: require("@/assets/images/onboarding-image-1.png"),
+    description:
+      "Compete in weekly 1v1 matchups, build your lineup, and climb the ranked ladder!",
+  },
+  {
+    title: "Create your team 🎽",
+    image: require("@/assets/images/onboarding-image-2.png"),
+    description:
+      "Pick a team name, choose a logo, and make your team uniquely yours!",
+  },
+  {
+    title: "Choose your game mode ⚔️",
+    image: require("@/assets/images/onboarding-image-3.png"),
+    description:
+      "Pick a mode, queue up, and challenge opponents in daily matchups!",
+  },
+  {
+    title: "Draft your team 💰",
+    image: require("@/assets/images/onboarding-image-4.png"),
+    description:
+      "Build your lineup within the salary cap  and make every pick count for every week!",
+  },
+  {
+    title: "Power up your game 🚀",
+    image: require("@/assets/images/onboarding-image-5.png"),
+    description:
+      "Use special augments to boost your squad and outplay your opponent! Strategy is key!",
+  },
+  {
+    title: "Climb the ranked ladder 🏆",
+    image: require("@/assets/images/onboarding-image-6.png"),
+    description:
+      "Compete in the daily matchups, win your games, and climb the ranks to the top!",
+  },
+];
+
+const OnboardingScreen = () => {
+  return (
+    <ImageBackground
+      className="flex-1 bg-gray-950"
+      resizeMode="cover"
+      source={require("@/assets/images/onboarding-background.png")}
+    >
+      <SafeAreaView className="flex-1 items-center justify-between">
+        <StatusBar barStyle="light-content" />
+        <Carousel data={carouselData} />
+        <View className="flex w-full gap-4 px-8 pb-8">
+          <Pressable className="rounded-lg bg-purple-600 p-3">
+            <Text className="text-center font-clash font-semibold uppercase text-white">
+              Create an Account
+            </Text>
+          </Pressable>
+          <PressableLink
+            className="rounded-lg p-3"
```

Comment:
I noticed that the link is clickable outside of the actual text (like horizontally). I suggest using marginHorizontal: auto or `mx-auto` to deal with this.

Reply:
Cool that works. My bad, I just realized this was supposed to be a button. I can't really see the outline around the background of the button. That's more in-line with the Figma as well, but this is just a nitpick to me. I'm okay with either a link or a button